docs/openapi.json

```diff
@@ -1860,7 +1862,7 @@
                 "duration": {
                     "title": "Duration",
                     "description": "Task duration in seconds",
-                    "type": "number"
+                    "type": "string"
```

Comment:
Why is this changed from a number to a string?

Reply:
I'm not sure, this file is automatically generated

Reply:
Confirmed that this is actually the correct type - the 'duration' reported in the task results page is a string, e.g. "00:00:58.796552".

The description, however, is incorrect, and should be changed to something like "Task duration, in hh:MM:SS.ss format"

Reply:
I updated the description, now says: "Task duration, in %H:%M:%S.%f format"

This is the Python format from DATETIME_FORMAT setting